### DISCUSSION TITLE
Console in Layout

### DISCUSSION THREAD

**Discussion by martinzerty** [Textualize/rich#3042] [Q&A] [answered]
Hello,

I would like to know if it was possible to print a console in a layout on panel object.
For example, in https://github.com/Textualize/rich/blob/master/examples/fullscreen.py , is it possible to change the progress bar by a console. I could print 5 or 7 lines for example, according to the max height and delete the oldest lines when trying to add a new one but max height is reached.

Thanks for your help !

**Comment by willmcgugan** [comment, marked as answer]
Possible, but nothing builtin. For this kind of thing you would be better off using [Textual](https://github.com/Textualize/textual)

**Reply by martinzerty** [reply]
Ok, thanks for your quick answer !
